modules/lang-painless/spi/src/main/java/org/elasticsearch/painless/spi/WhitelistLoader.java

```diff
@@ -460,11 +460,11 @@ public static Whitelist loadFromResourceFiles(Class<?> resource, Map<String, Whi
 
         List<Object> annotations;
 
-        line = line.trim();
-
-        if (line.isEmpty()) {
+        if ("".equals(line.replaceAll("\\s+",""))) {
```

Comment:
This is `line.isBlank()` in master, should master be updated?

Reply:
isBlank isn't available in 8

Reply:
No, doesn't exist in 8.